### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by AsparagusEduardo**
## Description
With this merged, we're ready for 1.6.0

## **Discord contact info**
asparaguseduardo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,40 +2,148 @@
 
 ## What is the pokeemerald Expansion?
 
-The Pokeemerald Expansion is a collection of feature branches that can be integrated into existing [pokeemerald](https://github.com/pret/pokeemerald) projects.
+The pokeemerald Expansion is a feature branch meant to be integrated into existing Pokémon Emerald hacks based of pret's [pokeemerald](https://github.com/pret/pokeemerald) decompilation project. This is ***NOT*** a standalone romhack, and as such, most features will be unavailable and/or unbalanced if played as is.
```
Comment (mrgriffin) [inline README.md:5]:
```suggestion
The pokeemerald Expansion is a feature branch meant to be integrated into existing Pokémon Emerald hacks based off pret's [pokeemerald](https://github.com/pret/pokeemerald) decompilation project. This is ***NOT*** a standalone romhack, and as such, most features will be unavailable and/or unbalanced if played as is.
```
Or perhaps "… based off of pret's…"

File: README.md [hunk 2]
```diff
@@ -2,40 +2,148 @@
...
+    - Mid-turn speed recalculation.
+    - Quick Poké Ball selection in Wild Battles
+        - Press `R` to use last selected Poké Ball
+        - Hold `R` to change selection with the D-Pad.
+    - Faster battle intro
+        - Message and animation/cry happens at the same time
+    - Faster HP drain.
+    - Battle Debug menu.
+        - Accessed by pressing `Select` on the "Fight/Bag/Pokémon/Run" menu.
+    - Option to use AI flags in wild Pokémon battles
```
Comment (mrgriffin) [inline README.md:39]:
```suggestion
    - Option to use AI flags in wild Pokémon battles.
```

File: README.md [hunk 3]
```diff
@@ -2,40 +2,148 @@
...
-    - Hidden Abilities data (How to make them available is up to the user).
-- Items from newer Generations and updated item effects for battle and field use.
+        - Gen 8+ damaging moves are given power extrapolated from Gen 7.
+        - Gen 8+ status moves have no additional effects, like Healing Wish.
+    - Initial battle parameters
+        - Queueing stat boosts (aka, Totem Boosts)
+        - Setting Terrains.
+    - Mid-turn speed recalculation.
+    - Quick Poké Ball selection in Wild Battles
+        - Press `R` to use last selected Poké Ball
```
Comment (mrgriffin) [inline README.md:32]:
```suggestion
        - Press `R` to use last selected Poké Ball.
```

File: README.md [hunk 4]
```diff
@@ -2,40 +2,148 @@
...
+        - Hold `R` to change selection with the D-Pad.
+    - Faster battle intro
+        - Message and animation/cry happens at the same time
```
Comment (mrgriffin) [inline README.md:35]:
```suggestion
        - Message and animation/cry happens at the same time.
```

File: README.md [hunk 5]
```diff
@@ -2,40 +2,148 @@
...
+- Breeding
+    - Incense Baby Pokémon now happen automatically (configurable).
+    - Level 1 eggs.
+    - Poké Ball inheriting.
+    - Egg Move Transfer, including Mirror Herb.
+    - Nature inheriting 100% of the time with Everstone
+    - Gen6+ Ability inheriting.
+- Items from newer Generations
+    - ***Gen 6+ Exp. Share*** (configurable)
+    -
```
Comment (mrgriffin) [inline README.md:94]: Was something supposed to be here?

**Review by mrgriffin — COMMENTED** [review]
Some very minor formatting corrections. I wasn't sure if you wanted `.`s at the end of bullet points which have children, so I left those alone, but I think there's a mix of `.`s and non-`.`s for those.

Good to see the readme being updated! :)

File: README.md (hunk 5 above)
Comment (AsparagusEduardo) [reply README.md:94]: Oh right, I wanted to add what items were added, or at least items added that are missing their functionality for now.

Comment (AsparagusEduardo) [reply README.md]: Fixed.

**Pull request merged**